CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -84,7 +84,11 @@ <h2 class="definition-title definition-title--search-result">
 
                             <div class="tooltip" role="tooltip">
                                 {% for definition in preverb.definitions %}
-                                    <p class="preverb-breakdown__preverb-definition">{{ definition }}</p>
+                                    <p class="preverb-breakdown__preverb-definition">{{ definition.text }}
+                                        {% for source in definition.source_ids %}
+                                            <cite class="cite-dict--popup">{{ source }}</cite>
```

Comment:
This is better written as:

```suggestion
                                            <cite class="cite-dict cite-dict--popup">{{ source }}</cite>
```